pkg/kubelet/cm/qos_container_manager_linux.go

```diff
@@ -81,8 +81,8 @@ func (m *qosContainerManagerImpl) GetQOSContainersInfo() QOSContainersInfo {
 func (m *qosContainerManagerImpl) Start(getNodeAllocatable func() v1.ResourceList, activePods ActivePodsFunc) error {
 	cm := m.cgroupManager
 	rootContainer := m.cgroupRoot
-	if !cm.Exists(rootContainer) {
-		return fmt.Errorf("root container %v doesn't exist", rootContainer)
+	if err := cm.Validate(rootContainer); err != nil {
+		return fmt.Errorf("root container %v doesn't exist: %v", rootContainer, err)
```

Comment:
same here. The error message coming back from `Validate()` will already contain the error message, so the error "wrapper" message here should just be something like:

```go
return fmt.Errof("root container invalid: %v", err)
```